### PR TITLE
Finished header Component

### LINKED ISSUE #35: Info Area Component (title. name, banner, more info)

### PR DESCRIPTION
**Pull request by DiogoGeada**
closes #35

### CODE REVIEW

File: src/components/headerComponent.vue [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            <button><v-icon>mdi-information-outline</v-icon></button>
+        </div>
+    </header>
+</template>
+
+<script>
+export default {
+    name: "Header",
+    props: [ "id", "name", "slug", "logo", "banner" ],
+    data: () => ({}),
```
Comment (jespernpedersen) [inline src/components/headerComponent.vue]: Don't include data if you are not using it.

File: src/components/headerComponent.vue [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    methods: {
+        getBannerImage: function () {
+            if(!this.banner) return "";
```
Comment (jespernpedersen) [inline src/components/headerComponent.vue]: It should return a placeholder image instead if one isn't set.

File: src/components/headerComponent.vue [hunk 3]
```diff
@@ -0,0 +1,63 @@
+<template>
+    <header id="header">
```
Comment (jespernpedersen) [inline src/components/headerComponent.vue]: Don't use id header for a tag that is already semantically header. Just use header or give it another name that is more specific to your usage of it.

File: src/components/headerComponent.vue [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+#header .title-area h2{
+    display: inline;
+}
+
+#header .title-area button{
+    position: absolute;
+    margin-left: 10px;
+    margin-top: 6px;
+}
+
```
Comment (jespernpedersen) [inline src/components/headerComponent.vue]: Remove empty space

**Review by jespernpedersen — CHANGES_REQUESTED** [review]

File: src/components/Header.vue [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+</template>
+
+<script>
+export default {
+    name: "Header",
+    props: [ "id", "title", "slug", "logo", "banner" ],
+    data: () => ({
+        bannerPlaceholder: require("@/assets/images/banner-placeholder.jpg"),
+        isLogoValid: true,
+        bannerStyle: {}
```
Comment (jespernpedersen) [inline src/components/Header.vue]: Data prop not being used.

**Review by jespernpedersen — CHANGES_REQUESTED** [review]

**Review by akram0171 — APPROVED** [review]

**Pull request merged**